### ISSUE TITLE
Please push a new release to PyPI

### ISSUE THREAD

**Issue by jikquantopian** [arngarden/MongoDBProxy#27]
There have been a lot of changes since the last PyPI release of this package. Would be great to get the current version on PyPI so we can install from there rather than from a URL. Thanks!

**Comment by arngarden** [comment]
Yes that would have been a good idea! Unfortunately I'm not in control of the PyPI release of the package. Not sure who has put it up there, but I'll do some digging..

**Comment by reinoldus** [comment]
Any update on this? Would it be an option for you to just make a new one and call it differently? Pypi seems to have a process to claim a repo, but seems to be quite complicated: https://stackoverflow.com/questions/20732179/pypi-can-i-claim-to-be-the-new-maintainer-of-an-unmaintained-package

**Comment by MiniXC** [comment]
As a temporary workaround, you can also install this package using
``pip install git+git://github.com/arngarden/MongoDBProxy.git#egg=mongodbproxy``

**Comment by arngarden** [comment]
Sorry for the delay! I've created a new PyPI package: https://pypi.org/project/MongoDBProxy-official/
